pom.xml

```diff
@@ -149,6 +149,11 @@
             <artifactId>annotations</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
Can we make this dependency optional? I am worried that this will lead in a wave of dependency breakages in various consumer projects.

Reply:
Instead of passing around a nullable value, I used ```java.util.Optional```.